lib/drivers/animasDriver.js

```diff
@@ -1131,6 +1197,33 @@ module.exports = function (config) {
     });
   };
 
+  var readTimeAdjust = function(cb) {
+    if (modelNumber === 'IR1295') {
```

Comment:
Will do, as it's also easier to read PING and VIBE instead of 'IR1295' and 'IR1285'.